boards/espressif/esp32_devkitc_wrover/doc/index.rst

```diff
@@ -1,6 +1,6 @@
 .. _esp32_devkitc_wrover:
 
-ESP32-DEVKITC-WROVER
+ESP32-DevKitC-WROVER
```

Comment:
I think you'll want a more global/systematic search and replace to address other offending spellings of all these board's names.
See line 53 for just one example, or L59.

Reply:
Sure, will update accordingly.